Tests/MeshLib/TestBoundaryElementSearch.cpp

```diff
@@ -175,3 +176,68 @@ TEST_F(MeshLibBoundaryElementSearchInSimpleHexMesh, SurfaceSearch)
         delete p;
 }
 
+// This is identical to the above
+// MeshLibBoundaryElementSearchInSimpleHexMesh.SurfaceSearch test but creates a
+// quadratic mesh from the original hex mesh.
```

Comment:
Maybe then extract the common part of the tests to some function?

Reply:
Some restructuring happened, though not very generalized, but code duplication at the top level is avoided.